src/Mvc/Mvc.Razor/src/RazorPageActivator.cs

```diff
@@ -32,7 +32,7 @@ public class RazorPageActivator : IRazorPageActivator
             IModelMetadataProvider metadataProvider,
             IUrlHelperFactory urlHelperFactory,
             IJsonHelper jsonHelper,
-            DiagnosticSource diagnosticSource,
+            DiagnosticListener diagnosticListener,
```

Comment:
This is a breaking change that we should not make.

Reply:
:(

Reply:
Changed back